mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/ListMailboxAssert.java

```diff
@@ -0,0 +1,105 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.store.mail.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.apache.james.mailbox.model.MailboxId;
+
+import com.google.common.base.Function;
+import com.google.common.base.Objects;
+import com.google.common.collect.FluentIterable;
+
+public class ListMailboxAssert {
+
+    private final List<Mailbox> actual;
+
+    private final List<InnerMailbox> mailboxtoInnerMailbox(List<Mailbox> mailboxes) {
+        return FluentIterable.from(mailboxes).transform(new Function<Mailbox, InnerMailbox>() {
+            @Override
+            public InnerMailbox apply(Mailbox input) {
+                return new InnerMailbox(input.getMailboxId(), input.getUser(), input.getName(), input.getNamespace());
+            }
+            
+        }).toList();
+    }
+
+    private ListMailboxAssert(List<Mailbox> actual) {
+        this.actual = actual;
+    }
+
+    public static ListMailboxAssert assertMailboxes(List<Mailbox> actual) {
+        return new ListMailboxAssert(actual);
+    }
+
+    public void containOnly(List<Mailbox> expecteds) {
```

Comment:
this one should be containsOnlyElementsOf and you should define a containsOnly with takes a vararg to make calling easier